biz.aQute.remote/test/biz/aQute/remote/AgentTest.java

```diff
@@ -186,8 +187,10 @@ public void testAgentUpdateBundleFromURL() throws Exception {
 	public void testAgentSupervisorTimeout() throws Exception {
 		TestSupervisor testSupervisor = new TestSupervisor();
 
-		testSupervisor.connect("localhost", 12345, 300);
-		testSupervisor.connect("localhost", 12345, 100);
+		try {
+			testSupervisor.connect("localhost", 12345, 300);
+			testSupervisor.connect("localhost", 12345, 100);
```

Comment:
You should put each connect call in its own try/catch block to make sure they both throw an exception.